jep/report-jep-conformance

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+REL_SCRIPT_DIR="`dirname \"$0\"`"
+SCRIPT_DIR="`( cd \"$REL_SCRIPT_DIR\" && pwd )`"
+PROJECT_ROOT="`( cd \"$SCRIPT_DIR/..\" && pwd )`"
+
+# `report-jep-conformance <jep-number>`
```

Comment:
Make this part of a `Jenkinsfile` so submitters do not forget?

Reply:
@jglick This repository does not have that yet.